### ISSUE TITLE
[Mobile-Android]Bottom of the page is blank when we open keypad and menu at the same time.

### ISSUE THREAD

**Issue by nivetheatha** [FightPandemics/FightPandemics#1209]
Environment : Staging
Steps to create:
1.Login with valid username and password.
2.Go to Help board.
3.Try to comment on the post.
4. Keypad  will open.
5.Open the menu at the same time.
6.Menu cut off at the bottom.
![Screenshot_20200717-125141_Chrome](https://user-images.githubusercontent.com/41587211/87822551-a7af4d80-c83f-11ea-888e-2ed92dd54fd3.jpg)

**Comment by OzkanAbdullahoglu** [comment]
I don't think that it is about opening the keypad, the menu also is cutting off at the bottom and able to scroll, am I missing something here? Also May I learn that which device is this snapshot coming from?

**Comment by nivetheatha** [comment]
@OzkanAbdullahoglu yes able to scroll it.If we open keypad simultaneously with menu the screen looks like the above picture. If we open menu only then it looks fine.I don't know exactly it happens because of keypad. 
I am using Samsung Galaxy s8,Google chrome.

**Comment by OzkanAbdullahoglu** [comment]
I can't reproduce this error in Iphone6 even in safari or chrome. Maybe you guys can reproduce? @idjevm @nr431207
